packages/reports/addon/components/common-actions/schedule.js

```diff
@@ -142,6 +153,29 @@ export default Ember.Component.extend({
       }
     },
 
+    onDelete() {
+      let deliveryRule = get(this, 'localDeliveryRule'),
+        deletePromise = new RSVP.Promise((resolve, reject) => {
+          this.attrs.onDelete(deliveryRule, { resolve, reject });
+        });
+
+      return deletePromise
+        .then(() => {
+          get(this, 'naviNotifications').add({
```

Comment:
Can we add a comment to say `Add page notification`, just to differentiate between this and the `set('notification')`?